### ISSUE TITLE
Form field placements off

### ISSUE THREAD

**Issue by luke-rogers** [stephanrauh/ngx-extended-pdf-viewer#2343]
**Describe the bug**
Open the attached PDF and notice the form field placements are off:
- Fields slightly overlap
- Radios are off centre

The attached PDF renders perfectly when tested on the [Mozilla viewer](https://mozilla.github.io/pdf.js/web/viewer.html).

**Version info**
20.0.2

**Demo PDF file**
[Hot Works Permit Template.pdf](https://github.com/stephanrauh/ngx-extended-pdf-viewer/files/15406455/Hot.Works.Permit.Template.pdf)

**Comment by stephanrauh** [comment]
Looks like a CSS problem. Do you know whether it looked better in previous versions?

**Comment by luke-rogers** [comment]
We were on v18 before and it was fine. We jumped to v20 so can't say whether it's reproducible on v19.

Happy to test if that would be useful?

**Comment by stephanrauh** [comment]
No, don't bother. Your information already helps me to find the missing CSS. Now I know I don't have to dig infinitely into the past.

**Comment by luke-rogers** [comment]
I think I've managed to track it down to [this line](https://github.com/stephanrauh/ngx-extended-pdf-viewer/blob/dbceda9e9d78aedf6ec6f7ea663c20ea65400dbd/projects/ngx-extended-pdf-viewer/src/lib/theme/common/ngx-extended-pdf-viewer.component.scss#L87) of CSS

Turn it off and the field returns to what I would expect.
<img width="509" alt="image" src="https://github.com/stephanrauh/ngx-extended-pdf-viewer/assets/8382642/9e0cb559-9305-4a46-b6cd-3728dbf422b5">

**Comment by stephanrauh** [comment]
Now that's crazy: pdf.js is using both box-sizing models. I'd call it an antipattern. Quite surprising given that the quality of the library is generally exceptionally good.

Be that as it may, I've applied a patch. It might be temporary - I hope I can find a better solution - but for now it solves both your issue and #2282.

Your bug fix has landed with version 20.1.0.

Enjoy!
Stephan

**Comment by luke-rogers** [comment]
The fix is working great - thanks for the quick turn around!
